pineappl_py/src/grid.rs

```diff
@@ -52,7 +52,7 @@ impl From<PyPidBasis> for PidBasis {
 
 // TODO: should probably be in a different module
 // TODO: rename to `OperatorSliceInfo`
```

Comment:
```suggestion
// TODO: should probably be in a different module
```
re other module - see my comment there, we could also shift - just decide something

Reply:
shouldn't we move the class into the `evolution` module?

Reply:
Yes, I think you are right. `OperatorSliceInfo` should be in the `evolution` module as it is a characteristic of the evolution. I will do so.

Reply:
Please check if you are now fine with [fc6d466](https://github.com/NNPDF/pineappl/pull/302/commits/fc6d4667b36df601e850691b59b0569daa4d90b4).